hera_pspec/uvwindow.py

```diff
@@ -168,6 +168,78 @@ def from_file(cls, ftfile, spw_range=None, verbose=False, x_orientation=None):
         return cls(data=ft_beam, pol=pol, freq_array=freq_array,
                    mapsize=mapsize, verbose=verbose, x_orientation=x_orientation)
  
+    @classmethod
+    def gaussian(cls, freq_array, widths, pol, 
+                 mapsize=1.0, npix=301,
+                 cosmo=conversions.Cosmo_Conversions(),
+                 verbose=False, x_orientation=None):
```

Comment:
For all the parameters that are literally just passed through to the main constructor, I would just set them as `**kwargs` and point in the docs to the main constructor's documentation.